src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Loader } from '../Loader/Loader';
+
+type Props = {
+  filteredTodos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ filteredTodos }) => (
+  <section className="todoapp__main" data-cy="TodoList">
+    {filteredTodos.map((todo) => {
+      const {
+        id,
+        title,
+        completed,
+      } = todo;
+
+      return (
```

Comment:
Extract `Todo` to separate component

Reply:
All the comments have been fixed in the 2-d part of the task: https://github.com/mate-academy/react_todo-app-add-and-delete/pull/128/commits/d0000616714059ac186d711e2cab6163cd3b13d3

Reply:
@iryna-borniak When you will have approved code for 2nd part - refactor this part too (just replace files). Then ask for review again.